### ISSUE TITLE
% Value on epics

### ISSUE THREAD

**Issue by WullaWullaWomba** [Kyusung4698/PoE-Overlay#119]
Hello once more,
If possible I'd like a value shown on epics that shows/indicates how well they rolled on variable mods

roughly like indicated below:
5 max life (1-10) - 44%
5 max mana (1-20) 21%
(44%+21%)/2 =34%

**Comment by Kyusung4698** [comment]
Seems like a nice addition. Ranges defined by Tiers #35? Or general mod range?

**Comment by WullaWullaWomba** [comment]
I was thinking the ranges are defined by the unique - dunno if you can check if an item is unique -> check it's ranges -> do the math?

**Comment by Kyusung4698** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Yes, but this is waiting for following issue: #35 Once this is in place this should be no problem to be implemented.

**Comment by C3ntraX** [comment]
Calculation like #420 would be nice :)
